test/unit/mocked_data/test_get_bgp_neighbors/normal/expected_result.json

```diff
@@ -4,7 +4,7 @@
     "peers": {
       "192.168.56.4": {
         "is_enabled": true,
-        "uptime": 1452696202,
```

Comment:
Yes, the "..." is just a workaround for results that may vary depending on when/where the test was run. This is a perfect example. @mirceaulinic check "get_facts" test case for another one.